news/10004.bugfix.rst

```diff
@@ -0,0 +1 @@
+- Annotate "typing.List" into "tools.tox_pip.pip()" (http://github.com/pypa/pip/pull/10004)
```

Comment:
This is not a user-facing change. Use `.trivial` instead.

Reply:
Got it. I will change it in a few minutes.

Reply:
I renamed `10004.bugfix.rst` to `10004.trivial.rst`. Was that ok?